nuxt.config.ts

```
@@ -160,6 +160,18 @@ export default defineNuxtConfig({
     baseUrl: URL,
   },
 
+  vite: {
+    optimizeDeps: {
+      include: ['@vueuse/core', 'zod', 'leaflet'],
+    },
+  },
```

Biome (2.5.1)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)